WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2365,11 +2365,6 @@ function(parent, parentElement, adminUrl) {
 
     menu.createSeparator();
 
-	if (!appCtxt.isExternalAccount() && appCtxt.get(ZmSetting.WEBCLIENT_OFFLINE_ENABLED)) {
-        mi = menu.createMenuItem("offlineSettings", {text: ZmMsg.offlineSettings});
-        mi.addSelectionListener(new AjxListener(this, this._offlineSettingsListener));
```

Comment:
we should remove _offlineSettingsListener listener also and all ZmMsg keys used inside that

Reply:
Removed